docs/Design/SoftDetailedDes/MIS.tex

```diff
@@ -242,6 +242,138 @@ \subsubsection{Local Functions}
 
 \newpage
 
+\section{MIS of Classification Module} \label{mClassification}
+
+The Classification Module is responsible for interpreting and identifying recognized items from receipt data.
+
+\subsection{Module}
+
+classificationModule
+
+\subsection{Uses}
+
+Database Driver Module (Section \ref{mDBDriver})
+
+\subsection{Syntax}
+
+\subsubsection{Exported Constants}
+
+None
+
+\subsubsection{Exported Types}
+
+\textbf{ClassifiedItem}
+
+\begin{table}[H]
```

Comment:
Can you please you primitive types here? I think I gave you the primitives in the previous comment I left regarding ClassifiedItem at the bottom or at least something it could look like. But stuff like, it wouldn't be double, it would be a real number ($\mathbb{R}) or String instead of VARCHAR

Reply:
resolved